manifests/0000_70_cluster-network-operator_01_pki_crd.yaml

```diff
@@ -1,12 +1,12 @@
-
----
-apiVersion: apiextensions.k8s.io/v1beta1
+# This file is automatically generated. DO NOT EDIT
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Not sure who to ask, but do you also want to update the CRD in https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/openshift-sdn/001-crd.yaml#L2 to `v1` as well? (from https://bugzilla.redhat.com/show_bug.cgi?id=1881636#c4, this BZ was originally for upgrading all the CRDs in openshift/api to v1)

If so, you may want to do it here and link this PR to that bugzilla

Reply:
This is just to fix codegen; let's make it a separate PR.